PSKoans/Koans/Constructs and Patterns/AboutCustomObjects.Koans.ps1

```diff
@@ -33,19 +33,19 @@ Describe '[PSCustomObject]' {
     It 'can be built by trimming objects down' {
         $Object = Get-ChildItem -Path $Home | Select-Object -First 1 -Property Name, Parent
         $Object | Should -BeOfType PSCustomObject
-        __ | Should -Be $Object.Parent.Name
+        '__' | Should -Be $Object.Parent.Name
     }
 
     It 'can have arbitrary properties' {
-        $Object = [PSCustomObject]@{ '__' = 'Enter Property Name' }
+        $Object = [PSCustomObject]@{ '__' = 'PropertyValue' }
 
-        __ | Should -Be $Object.PSObject.Properties.Count
-        __.PSObject.Properties.Name | Should -Be 'PropertyName'
+        '__' | Should -Be $Object.PSObject.Properties.Count
+        '__'.PSObject.Properties.Name | Should -Be 'PropertyName'
     }
 
     It 'can be added to' {
         $Object = [PSCustomObject]@{ 'Property1' = 12 }
-        $Object | Add-Member -MemberType NoteProperty -Name 'Property2' -Value __
+        $Object | Add-Member -MemberType NoteProperty -Name 'Property2' -Value '__'
```

Comment:
Same as above; the value we're expecting is numeric, so no quotes. 🙂 

There are a handful of others below that should also not have quotes.

Reply:
Fixed.